### PR TITLE
feat(k8sprocessor): expose delimiter to configuration for services

### PR DESCRIPTION
**Pull request by sumo-drosiek**
Signed-off-by: Dominik Rosiek <drosiek@sumologic.com>

### CODE REVIEW

File: pkg/processor/k8sprocessor/factory.go [hunk 1]
```diff
@@ -49,6 +49,7 @@ func createDefaultConfig() config.Processor {
 	return &Config{
 		ProcessorSettings: config.NewProcessorSettings(config.NewID(typeStr)),
 		APIConfig:         k8sconfig.APIConfig{AuthType: k8sconfig.AuthTypeServiceAccount},
+		Extract:           ExtractConfig{Delimiter: ", "},
```
Comment (pmalek-sumo) [inline pkg/processor/k8sprocessor/factory.go:52]: Can we name the default and use it (the defined `const`)?

Comment (sumo-drosiek) [reply pkg/processor/k8sprocessor/factory.go]: fixed

File: pkg/processor/k8sprocessor/factory.go [hunk 2]
```diff
@@ -49,6 +49,7 @@ func createDefaultConfig() config.Processor {
 	return &Config{
 		ProcessorSettings: config.NewProcessorSettings(config.NewID(typeStr)),
 		APIConfig:         k8sconfig.APIConfig{AuthType: k8sconfig.AuthTypeServiceAccount},
+		Extract:           ExtractConfig{Delimiter: DefaultDelimiter},
```
Comment (pmalek-sumo) [inline pkg/processor/k8sprocessor/factory.go:52]: how about?

```suggestion
		Extract:           ExtractConfig{
			Delimiter: DefaultDelimiter
		},
```

**Review by pmalek-sumo — APPROVED** [review]

File: pkg/processor/k8sprocessor/factory.go [hunk 3]
```diff
@@ -49,6 +49,9 @@ func createDefaultConfig() config.Processor {
 	return &Config{
 		ProcessorSettings: config.NewProcessorSettings(config.NewID(typeStr)),
 		APIConfig:         k8sconfig.APIConfig{AuthType: k8sconfig.AuthTypeServiceAccount},
+		Extract:           ExtractConfig{
+			Delimiter: DefaultDelimiter
```
Comment (sumo-drosiek) [inline pkg/processor/k8sprocessor/factory.go:53]:
```suggestion
			Delimiter: DefaultDelimiter,
```

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**